code/game/objects/items/storage/book.dm

```diff
@@ -225,6 +224,19 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 				EX.icon_state = "ghost1"
 				EX.name = "Purified [initial(EX.name)]"
 			user.visible_message("<span class='notice'>[user] has purified [SS]!</span>")
+	else if(istype(A, /obj/item/nullrod/scythe/talking))
+		var/obj/item/nullrod/scythe/talking/sword = A
+		to_chat(user, "<span class='notice'>You begin to exorcise [sword].</span>")
+		playsound(src,'sound/hallucinations/veryfar_noise.ogg',40,TRUE)
```

Comment:
very spammable sound

Reply:
you can do the same thing with a soulstone currently, so I figured that this'd be okay